packages/vscode-extension/src/extension/main.ts

```diff
@@ -192,23 +215,43 @@ function watchPlipluginFolder(
   // watch for folder create/delete events
   folderWatcher.onDidCreate(() => {
     client.sendNotification(WorkspaceDidChangePlipluginConfigNotification);
+    sendTelemetryEvent(
+      "pli.language.support.onDidCreate.folder",
+      telemetryReporter,
+    );
```

Comment:
The `folderWatcher` and `fileWatcher` only target the `.pliplugin` folder. Other folders are not watched. Is that the expected behavior?